x/auth/module.go

```diff
@@ -159,10 +163,16 @@ func (AppModule) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
 
 // RegisterStoreDecoder registers a decoder for auth module's types
 func (am AppModule) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
-	sdr[StoreKey] = simulation.NewDecodeStore(am.cdc)
+	sdr[StoreKey] = simulation.NewDecodeStore(am.accountKeeper)
 }
 
 // WeightedOperations doesn't return any auth module operation.
 func (AppModule) WeightedOperations(_ module.SimulationState) []simtypes.WeightedOperation {
 	return nil
 }
+
+// InterfaceModule is an interface that modules can implement in order to
+// register their interfaces and implementations in an InterfaceRegistry
+type InterfaceModule interface {
```

Comment:
q: why is this declared here? do we need to define this for every module?

Reply:
No, this is present in app level and isn't needed here. Thanks for spotting it out